packit.yaml

```diff
@@ -19,24 +19,13 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-    - fedora-40
-    - fedora-41
-    - fedora-latest-aarch64
-    - fedora-development
     - centos-stream-9-x86_64
-    - centos-stream-9-aarch64
-    - centos-stream-10
 
   - job: tests
     trigger: pull_request
     targets:
-      - fedora-40
-      - fedora-41
-      - fedora-latest-aarch64
-      - fedora-development
-      - centos-stream-9-x86_64
-      - centos-stream-9-aarch64
-      - centos-stream-10
+      centos-stream-9-x86_64:
+        distros: ["centos-stream-9", "CentOS-Stream-9-image-mode"]
 
```

Comment:
```suggestion
    tf_extra_params:
        environments:
          - artifacts:
              - type: repository
                id: https://my.repo/repository
```

Reply:
@martinpitt let's try this, this should disable the artifacts installation I hope :)

Reply:
@thrix Thanks for the hint! I'm a bit sceptical, as in [cockpit](https://github.com/cockpit-project/cockpit/blob/main/packit.yaml#L46) that felt additive, not replacing. But pushed, let's try!

Reply:
@thrix No, didn't help unfortunately: https://artifacts.dev.testing-farm.io/5ba18f34-5551-42e1-867f-16d24984ee20/

Reply:
Reported as https://issues.redhat.com/browse/TFT-3028

Reply:
No need to report that, we have other ways to try it out, let me open a new MR so i can experiment and help

Reply:
continuing myself in !1924 for a prototype